### PR TITLE
More better hero slider with separate divs (finally)

### PR DESCRIPTION
**Pull request by strikemike2k**
I'm pretty happy with this solution. No more having to deal with different size images scaling during the transition.

### CODE REVIEW

File: page-templates/home.php [hunk 1]
```diff
@@ -28,24 +28,75 @@
...
+    $('.hero-slider').on('afterChange', (event, slick, currentSlide) => updateHeroImage(slick, currentSlide));
   });
+
+  function drawHeroImageContainers(slick) {
+    $(heroImages).get().reverse().forEach((image, index) => {
+      $(slick.$list[0]).prepend(
+        $('<div></div>')
+          .addClass('hero-image')
+          .css('background-image', 'url("' + image.src + '")')
+          .css('background-position', 'center center')
```
Comment (shadoath) [inline page-templates/home.php]: This will be where our custom field for background position will be applied?

File: page-templates/home.php [hunk 2]
```diff
@@ -28,24 +28,75 @@
...
+  }
+
+  function updateHeroImage(slick, currentSlide, force = false) {
+    const nextSlide = Math.max(0, Math.min(slick.slideCount - 1, currentSlide));
+    if(nextSlide != currentHeroSlide || force) {
+      currentHeroSlide = nextSlide;
+      const images = $('.hero-image');
+      $(images).fadeOut(1000);
+      $(images[currentSlide]).fadeIn(1000);
+      $(slick.$slides).removeClass('slick-current');
```
Comment (shadoath) [inline page-templates/home.php]: what is the magic in this `slick.$slides?`

**Comment by jreynolds13** [comment]
https://www.youtube.com/watch?v=Ip1SYl97kh4 (SFW)

File: page-templates/home.php (hunk 1 above)
Comment (strikemike2k) [reply page-templates/home.php]: Correct

File: page-templates/home.php (hunk 2 above)
Comment (strikemike2k) [reply page-templates/home.php]: `slick.$slides` returns an array of the slide elements. So I am using jQuery to remove the slick-current class and later add it to the selected slide. This is required for the custom slide navigation.

**Pull request merged**